include/box_data_type.h

```diff
@@ -5,7 +5,6 @@
 #include <stdbool.h>
 #include <string>
 
-typedef float float32_t;
 typedef double float64_t;
```

Comment:
there is no need to define float64_t since double is always double precision floating point number under IEEE754

Reply:
this was defined to be aligned with int32_t etc...

Reply:
I do not think we should define float64_t just for the sake of name consistency.

Reply:
fixed